src/mame/drivers/cninja.cpp

```diff
@@ -1621,12 +1621,12 @@ ROM_START( edrandy ) /* World ver 3 */
 	ROM_LOAD( "mad-02",   0x000000, 0x80000, CRC(6c76face) SHA1(e485b118e1e5bdf130c7ae29eea2f192f85f93a5) ) /* tiles 3 */
 
 	ROM_REGION( 0x500000, "gfx4", 0 )
-	ROM_LOAD( "mad-05",   0x000000, 0x80000, CRC(3f2ccf95) SHA1(ed9585f2162ca53a7621b86a9def45a46cd08331) )
+	ROM_LOAD( "mad-05",   0x000000, 0x80000, CRC(3f2ccf95) SHA1(ed9585f2162ca53a7621b86a9def45a46cd08331) ) /* sprites */
```

Comment:
Should go to the top, not to the right.
Or, even better: all _gfx(\d+)_ tags are legacy, renaming them all to a meaningful name and ditch the comment altogether is preferred.